FreeRTOS/iot-reference-nxp-rt1060
# Update MQTT stream submodule pointer

Description
-----------
In this PR:
* Update MQTT stream submodule pointer
* Add MQTTFileDownloader_config.h file to every project which make use of MQTT stream.
* Onboard manifest check

Test Steps
-----------
Run the OTA demo without problem.

Checklist:
----------



Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
